template/azurePipelines/kustomize/draft.yaml

```diff
@@ -0,0 +1,30 @@
+variables:
+  - name: "PIPELINE_NAME"
+    default:
+      value: "Build and deploy an app to AKS"
+    description: "the name of the azure pipeline"
+  - name: "BRANCH_NAME"
+    default:
+      value: "main"
+    description: "the branch to trigger the pipeline"
+  - name: "ARM_SERVICE_CONNECTION"

```

Comment:
this is really just the azure subscription right? Renaming might be good

Reply:
No, it's the ADO service connection that is linked to an msi to access the AKS cluster and ACR